code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -223,8 +222,7 @@
 	SET_REACTION_RESULTS((plasma_burn_rate) * (1 + oxygen_burn_ratio))
 	var/energy_released = FIRE_PLASMA_ENERGY_RELEASED * plasma_burn_rate
 	var/new_heat_capacity = air.heat_capacity()
-	if(new_heat_capacity > MINIMUM_HEAT_CAPACITY)
-		air.temperature = (temperature * old_heat_capacity + energy_released) / new_heat_capacity
+	air.temperature = (temperature * old_heat_capacity + energy_released) / new_heat_capacity
```

Comment:
this code isn't hot enough to worry about the cost of this, and while it is currently an impossible to fail check, that's only because we assume a base heat cap for space. this may not always be the case so I don't think it's a good idea to rely on.